src/components/Routing.js

```diff
@@ -144,12 +155,24 @@ class Routing extends Component {
         return (
             <Container className='App' mainnet={IS_MAINNET}>
                 <GlobalStyle />
-                <ConnectedRouter basename={PATH_PREFIX}  history={this.props.history}>
+                <ConnectedRouter basename={PATH_PREFIX} history={this.props.history}>
                     <ThemeProvider theme={theme}>
                         <ScrollToTop/>
                         <NetworkBanner/>
                         <Navigation/>
                         <GlobalAlert/>
+                        {/* <EnableTwoFactorPromptModal/> */}
```

Comment:
is this some debugging stuff commented out?

Reply:
Related: https://github.com/near/near-wallet/pull/700/files/577d401fae66b2ec1ae621feb5b58802a03bd837#r457813385

Reply:
Will be removed

Reply:
@vgrichina this is the '2fa reminder' modal that we're gonna show to users that we believe should enable 2fa but haven't yet. @kcole16 is this something we should include for first push? And if so, any thoughts on the minimum balance required to trigger? And should there be a way for user to not be bothered again?